meetings/2023/2023-02-14.md

```diff
@@ -0,0 +1,88 @@
+# OpenJS Foundation Cross Project Council Meeting 2023-02-14
+
+## Links
+
+* **Recording**:
```

Comment:
Do we have the recording?

Reply:
@rginn told me it is recorded. When she uploads the file to Youtube, I'll get the URL and update here before merging